scripts/g.download.location/g.download.location.py

```diff
@@ -102,7 +102,9 @@ def main(options, unused_flags):
 
     if destination.exists():
         gs.fatal(
-            _("Location named <{}> already exists, download canceled").format(name)
+            _("Location named <{}> already exists in <{}>, download canceled").format(
+                name, destination
```

Comment:
With more than one substitution, you need to use keywords. For the message itself, I think "in" suggests it is inside a directory, so you should put there what is in the database variable, not destination. So:

```suggestion
            _("Location named <{name}> already exists in <{directory}>, download canceled").format(
                name=name, directory=database
```